ambry-server/src/main/java/com.github.ambry.server/ServerBlobStorageService.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2019 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.server;
+
+import com.github.ambry.commons.CopyingAsyncWritableChannel;
+import com.github.ambry.network.NettyServerRequest;
+import com.github.ambry.network.NettyServerRequestResponseChannel;
+import com.github.ambry.rest.BlobStorageService;
+import com.github.ambry.rest.RestRequest;
+import com.github.ambry.rest.RestResponseChannel;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+public class ServerBlobStorageService implements BlobStorageService {
```

Comment:
Can you state the purpose of the this BlobStorageService? 

I see in the handlePost method, it calls requestResponseChannel.sendRequest, which is just queuing up with RestRequest in a ArrayBlockingQueue. But if you see the logic in the AmbryBlobStorageService, in handleXXX, it already starts processing request, not just queuing it. I think either the name is confusing and needs to be changed, or something is not right here

Reply:
Did you mean to add this now?  It looks incomplete.  Please add javadoc.

Reply:
@justinlin-linkedin  It's a placeholder.